src/azurelinux/3.0/net10.0/cross/loongarch64/Dockerfile

```diff
@@ -3,9 +3,6 @@ ARG ROOTFS_DIR=/crossrootfs/loongarch64
 FROM mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-crossdeps-builder-amd64 AS builder
 ARG ROOTFS_DIR
 
-# temporarily bypass cache to fetch the latest arcade version with recent fixes
-RUN rm -rf /scripts && git clone --depth 1 --single-branch https://github.com/dotnet/arcade /scripts
```

Comment:
`src/azurelinux/3.0/net10.0/crossdeps-builder/amd64/Dockerfile` is rebuilding which will pull the new arcade. Rebuilding all cross/ images with llvm requires touching `src/azurelinux/3.0/net10.0/crossdeps-llvm/amd64/Dockerfile`, see https://github.com/dotnet/dotnet-buildtools-prereqs-docker/issues/1312.